lib/ansible/modules/unarchive.py

```diff
@@ -856,6 +863,14 @@ def main():
     except Exception as e:
         module.fail_json(msg="Source '%s' not readable, %s" % (src, to_native(e)))
 
+    # create dest directory if create_dir is true
+    if create_dir:
+        try:
+            if not os.path.exists(b_dest):
+                os.makedirs(b_dest)
```

Comment:
https://docs.python.org/3/library/os.html#os.makedirs
It has been introduced in python 3.2 which is fine as https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#control-node-requirements has said requirements of python 3 is upper than 3.5.
But I'm not sure how to deal with it as python 2.7 is still supported too.